models/classes/class.LtiUser.php

```diff
@@ -51,7 +53,7 @@ class taoLti_models_classes_LtiUser
 	
 	public function __construct(taoLti_models_classes_LtiLaunchData $ltiLaunchData) {
 	    $this->ltiLaunchData = $ltiLaunchData;
-	    $this->userUri = taoLti_models_classes_LtiService::singleton()->findOrSpawnUser($ltiLaunchData)->getUri();
+	    $this->userUri = $this->getServiceManager()->get(AbstractLtiService::SERVICE_ID)->findOrSpawnUser($ltiLaunchData)->getUri();
```

Comment:
no calls to getServiceManager in __construct() as there was no opportunity for injection yet